### PR TITLE
fix: Use caret dependency range for Griffel

### PR DESCRIPTION
**Pull request by ling1726**

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                            |   Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                               Change |
| :----------------------------------------------------------------------------------------------------------- | -------------------------: | --------------------------: | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr> | `14.76 kB`<br />`5.596 kB` | `14.766 kB`<br />`5.597 kB` | `6 B` <br />`1 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                   |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                          |          `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr>          |          `554 B`<br />`348 B` |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr>            |    `2.936 kB`<br />`1.212 kB` |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr> |  `77.836 kB`<br />`23.683 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                             |   `21.653 kB`<br />`6.862 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                               |   `22.557 kB`<br />`7.172 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                             |   `23.357 kB`<br />`7.099 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                          |   `35.704 kB`<br />`9.499 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                          |  `42.777 kB`<br />`10.733 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                  |  `38.279 kB`<br />`10.365 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                |   `45.612 kB`<br />`11.71 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                              |  `50.929 kB`<br />`10.919 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                      |  `66.957 kB`<br />`19.042 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                |  `62.113 kB`<br />`17.911 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                    |    `8.461 kB`<br />`3.555 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                    |   `10.026 kB`<br />`4.067 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                  |     `8.562 kB`<br />`3.61 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>       |   `73.69 kB`<br />`23.901 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>       |  `73.203 kB`<br />`23.895 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                       |   `16.321 kB`<br />`5.837 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                             |    `10.68 kB`<br />`4.215 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                             |   `22.861 kB`<br />`7.512 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                             |    `9.238 kB`<br />`3.815 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                               |    `12.082 kB`<br />`4.88 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                     |   `10.839 kB`<br />`4.146 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                       | `104.924 kB`<br />`31.964 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                          |   `10.033 kB`<br />`3.753 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                          |   `23.839 kB`<br />`8.292 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                             |   `30.438 kB`<br />`10.41 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                   |   `14.319 kB`<br />`5.711 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                          |   `20.002 kB`<br />`7.171 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                          |   `26.136 kB`<br />`8.387 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                              |  `43.378 kB`<br />`12.277 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                       |   `19.073 kB`<br />`6.308 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                          |     `26.578 kB`<br />`8.6 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                              |   `11.572 kB`<br />`4.537 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                            |   `14.882 kB`<br />`4.977 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                    |   `22.228 kB`<br />`7.523 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>              |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                        |   `28.995 kB`<br />`6.215 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                      |    `16.973 kB`<br />`4.86 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                       |   `44.55 kB`<br />`15.386 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                             |          `189 B`<br />`161 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d8997f68c68e6c71a09f9db87ec54aff9cd9a651'>d8997f68c68e6c71a09f9db87ec54aff9cd9a651</a></sub>
<div id="bundleSizeComment9423"/>

**Review by spmonahan — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/Avatar-mount_base.html">1386</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/Avatar-mount.html">1330</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/Button-mount_base.html">1038</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/Button-mount.html">1011</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProvider-mount_base.html">1799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProvider-mount.html">1781</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProviderWithTheme-mount_base.html">750</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProviderWithTheme-mount.html">725</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">622</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">740</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/MakeStyles-mount_base.html">2057</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test-react-components/MakeStyles-mount.html">1966</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by tudorpopams — APPROVED** [review]

**Review by micahgodbolt — APPROVED** [review]

**Review by TristanWatanabe — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/BaseButton-mount_base.html">789</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/BaseButton-mount.html">790</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Breadcrumb-mount_base.html">2268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Breadcrumb-mount.html">2269</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Checkbox-mount_base.html">1248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Checkbox-mount.html">1270</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/CheckboxBase-mount_base.html">1064</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/CheckboxBase-mount.html">1072</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ChoiceGroup-mount_base.html">3905</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ChoiceGroup-mount.html">3875</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ComboBox-mount_base.html">766</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ComboBox-mount.html">839</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/CommandBar-mount_base.html">8990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/CommandBar-mount.html">8882</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ContextualMenu-mount_base.html">9767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ContextualMenu-mount.html">9611</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DefaultButton-mount_base.html">949</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DefaultButton-mount.html">955</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DetailsRow-mount_base.html">3027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DetailsRow-mount.html">3034</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DetailsRowFast-mount_base.html">3093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DetailsRowFast-mount.html">3045</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DetailsRowNoStyles-mount_base.html">2879</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DetailsRowNoStyles-mount.html">2882</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Dialog-mount_base.html">2360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Dialog-mount.html">2383</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DocumentCardTitle-mount_base.html">143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/DocumentCardTitle-mount.html">144</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Dropdown-mount_base.html">2827</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Dropdown-mount.html">2800</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/FocusTrapZone-mount_base.html">1580</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/FocusTrapZone-mount.html">1543</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/FocusZone-mount_base.html">1513</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/FocusZone-mount.html">1501</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/IconButton-mount_base.html">1380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/IconButton-mount.html">1409</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Label-mount_base.html">326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Label-mount.html">325</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Layer-mount_base.html">2542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Layer-mount.html">2540</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Link-mount_base.html">425</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Link-mount.html">437</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/MenuButton-mount_base.html">1219</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/MenuButton-mount.html">1231</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/MessageBar-mount_base.html">1839</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/MessageBar-mount.html">1839</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Nav-mount_base.html">2636</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Nav-mount.html">2601</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/OverflowSet-mount_base.html">973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/OverflowSet-mount.html">1003</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Panel-mount_base.html">1842</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Panel-mount.html">1849</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Persona-mount_base.html">830</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Persona-mount.html">837</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Pivot-mount_base.html">1143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Pivot-mount.html">1137</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/PrimaryButton-mount_base.html">1090</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/PrimaryButton-mount.html">1090</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Rating-mount_base.html">6636</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Rating-mount.html">6618</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SearchBox-mount_base.html">1105</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SearchBox-mount.html">1124</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Shimmer-mount_base.html">2207</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Shimmer-mount.html">2234</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Slider-mount_base.html">1659</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Slider-mount.html">1685</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SpinButton-mount_base.html">3939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SpinButton-mount.html">3970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Spinner-mount_base.html">405</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Spinner-mount.html">406</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SplitButton-mount_base.html">2499</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SplitButton-mount.html">2446</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Stack-mount_base.html">483</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Stack-mount.html">488</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/StackWithIntrinsicChildren-mount_base.html">1887</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/StackWithIntrinsicChildren-mount.html">1840</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/StackWithTextChildren-mount_base.html">4651</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/StackWithTextChildren-mount.html">4642</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SwatchColorPicker-mount_base.html">9011</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/SwatchColorPicker-mount.html">9073</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/TagPicker-mount_base.html">2013</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/TagPicker-mount.html">2010</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/TeachingBubble-mount_base.html">72938</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/TeachingBubble-mount.html">74063</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Text-mount_base.html">401</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Text-mount.html">395</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/TextField-mount_base.html">1152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/TextField-mount.html">1153</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ThemeProvider-mount_base.html">949</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ThemeProvider-mount.html">943</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ThemeProvider-virtual-rerender_base.html">609</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ThemeProvider-virtual-rerender.html">599</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1428</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Toggle-mount_base.html">696</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/Toggle-mount.html">699</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/buttonNative-mount_base.html">99</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23754/perf-test/buttonNative-mount.html">105</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: yarn.lock [hunk 1]
```diff
@@ -1657,7 +1657,7 @@
   resolved "https://registry.yarnpkg.com/@fluentui/react-icons/-/react-icons-2.0.172-rc.8.tgz#5daa62d1b11df87cdf26cbfd5e3aa7328e367e74"
   integrity sha512-vXhOcnlsYA/XgiaiPo4Cyp5GeI+CZE2zRVBNLkygemrafKnboLwUh9dLb/qC4YLsQ4DTaYPrvFq4cKujXBhl4w==
 
-"@griffel/babel-preset@1.3.2", "@griffel/babel-preset@^1.3.2":
+"@griffel/babel-preset@^1.3.2":
```
Comment (layershifter) [inline yarn.lock:1660]: This went wild, it seems that something was not bumped properly

Comment (ling1726) [reply yarn.lock:1660]: good catch 👍 I deduped, and now the only version is 1.3.3

**Pull request merged**